bin/turbo_rspec

```diff
@@ -54,23 +56,54 @@ OptionParser
     opts.on("--use-runtime-info", "Use runtime info for tests group splitting") do
       use_runtime_info = true
     end
+
+    opts.on("--enable-system-tests", "Run the system tests (defaults false)") do
+      enable_system_tests = true
+    end
+
+    opts.on("--disable-system-tests", "Don't run the system tests (defaults true)") do
+      disable_system_tests = true
+    end
   end
   .parse!(ARGV)
 
-requires.each { |f| require(f) }
-
-formatters << { name: "progress", outputs: [] } if formatters.empty?
 
-formatters.each { |formatter| formatter[:outputs] << "-" if formatter[:outputs].empty? }
+if enable_system_tests && disable_system_tests
+  STDERR.puts "Error: I'm not sure how to enable and disable system tests simultaneously"
+  exit 1
+end
 
+# OptionParser modifies ARGV, leaving the leftover arguments in ARGV
 if ARGV.empty?
-  files = TurboTests::Runner.default_spec_folders
+  if !enable_system_tests && !disable_system_tests
+    STDERR.puts "Not running system tests since it wasn't explicitly specified"
+  end
+
+  run_system_tests = !!enable_system_tests
+
+  files =
+    if run_system_tests
+      ["#{Rails.root}/spec"]
+    else
+      TurboTests::Runner.default_spec_folders
+    end
+
   use_runtime_info = true if use_runtime_info.nil?
 else
+  if enable_system_tests || disable_system_tests
+    STDERR.puts "Ignoring system test options since files were specified"
+  end
```

Comment:
I'm not sure how helpful this error message is... if you're specifying a system test file, this may be confusing?

Reply:
Can you think of a better way to phrase it? If you do `bin/turbo_rspec --enable-system-tests spec/system`, I want to warn that the `--enable-system-tests` option has no effect.

Reply:
For debugging, it's fine as is. Once we have just one of the two flags, the message could read: 

`The --enable/disable-system-tests flag has no effect when specifying files`